### PR TITLE
Add CheckboxFormField

### PR DESCRIPTION
**Pull request by Hanashi**
This PR adds an implementation of a checkbox form field for boolean values to the WSC.

### CODE REVIEW

File: wcfsetup/install/files/lib/system/form/builder/field/CheckboxFormField.class.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	
+	/**
+	 * @inheritDoc
+	 */
+	public function getHtml() {
+		return WCF::getTPL()->fetch(
+			$this->templateName,
+			'wcf',
+			[
+				'field' => $this
```
Comment (Gravatronics) [inline wcfsetup/install/files/lib/system/form/builder/field/CheckboxFormField.class.php:28]:
```suggestion
				'field' => $this,
```

**Review by ghost — CHANGES_REQUESTED** [review]
Please add `__checkboxFormField` at the alphabetical correct position in `syncTemplates.json`.

File: wcfsetup/install/files/lib/system/form/builder/field/CheckboxFormField.class.php (hunk 1 above)
Comment (Hanashi) [reply wcfsetup/install/files/lib/system/form/builder/field/CheckboxFormField.class.php]: I added the missing comma

**Comment by Hanashi** [comment]
I added `__checkboxFormField` to `syncTemplates.json` and added the missing comma to `CheckboxFormField.class.php`.

**Pull request merged**